pkg/validation/validate_cluster.go

```diff
@@ -96,6 +109,41 @@ func getRoleNode(node *v1.Node) string {
 	return role
 }
 
+func collectComponentStatus(client k8s_clientset.Interface) (bool, error) {
+	componentList, err := client.CoreV1().ComponentStatuses().List(v1.ListOptions{})
+	if err != nil {
+		fmt.Println("first error")
```

Comment:
remove or switch to `glog.V(8)` or something if there's something important.  You might want to wrap the error message because it's coming from external code, but I don't think it much matters here.